### PR TITLE
MEN-1420 docs: add spec for GET /limits/max_devices

### PR DESCRIPTION
**Pull request by bboozzoo**
@mendersoftware/rndity @maciejmrowiec

### CODE REVIEW

File: docs/management_api.yml [hunk 1]
```diff
@@ -175,6 +175,32 @@ paths:
...
+          format: Bearer [token]
+          description: |
+            Contains the JWT token issued by the User Administration and
+            Authentication Service.
+      responses:
+        200:
+          description: Usage statistics and limits.
+          schema:
+            $ref: '#/definitions/Limit'
+        404:
```
Comment (PiotrPrzybylak) [inline docs/management_api.yml]: Do we need 404 here?

Comment (bboozzoo) [reply docs/management_api.yml]: removed

**Comment by maciejmrowiec** [comment]
what about the usage ?

**Comment by bboozzoo** [comment]
AFAIU the task is only about limits: https://tracker.mender.io/browse/MEN-1420

**Pull request merged**